src/styles/PointOfCareApp.scss

```diff
@@ -0,0 +1,71 @@
+@import './variables';
+
+.FullApp-content {
```

Comment:
We should really change this class and the associate className in the code to align with the POC naming convention.